### PR TITLE
Adding Telco release notes

### PR DESCRIPTION
**Pull request by ahardin-rh**

### CODE REVIEW

**Comment by ahardin-rh** [comment]
FYI @codyhoag @vikram-redhat @sheriff-rh :rocket:

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -410,6 +427,18 @@ Partial Tuned real-time profile support became available in {product-title} 4.4.
 Now, the real-time profiles are fully compatible with what the real-time
 profiles do in Tuned on {op-system-base-full}.
 
+[id="ocp-4-6-perforance-addon-operator"]
```
Comment (sheriff-rh) [inline release_notes/ocp-4-6-release-notes.adoc:430]: performance-addon-operator

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -255,6 +255,23 @@ Starting in {product-title} 4.6, the Red Hat-provided default catalogs used by O
 
 See xref:../release_notes/ocp-4-6-release-notes.adoc#ocp-4-6-operator-catalogs-per-version[Default Operator catalogs now shipped per cluster version] for more details and important Operator upgrade prerequisites.
 
+[id="ocp-4-6-deployment-without-a-provisioning-network"]
+==== Deployment without a provisioning network
+
+{product-title} now supports deployment without a provisioning network, and for RedFish Virtual Media.
```
Comment (sheriff-rh) [inline release_notes/ocp-4-6-release-notes.adoc:261]: Not sure a comma is necessary here

File: release_notes/ocp-4-6-release-notes.adoc [hunk 3]
```diff
@@ -410,6 +427,18 @@ Partial Tuned real-time profile support became available in {product-title} 4.4.
 Now, the real-time profiles are fully compatible with what the real-time
 profiles do in Tuned on {op-system-base-full}.
 
+[id="ocp-4-6-perforance-addon-operator"]
+==== The Performance Addon Operator is now fully supported
+
+The Performance Addon Operator helps the administrator with tuning worker nodes for low latency and real-time workloads. It takes a high level tuning intent in the form of `PerformanceProfile` custom resource and translates it into all the actions necessary to configure the Linux kernel, operating system, huge pages, and kubelet for low latency purposes.
```
Comment (sheriff-rh) [inline release_notes/ocp-4-6-release-notes.adoc:433]: high-level tuning intent in the form of a `PerformanceProfile` Custom Resource

File: release_notes/ocp-4-6-release-notes.adoc [hunk 4]
```diff
@@ -453,6 +482,28 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-secondary-device-metrics-network-attachments"]
+==== Secondary device metrics can be associated with network attachments
+
+Secondary devices, or interfaces, are used for different purposes. It is important to have a way to classify them so that you can aggregate the metrics for secondary devices with the same classification.
+
+The kubelet is already publishing a set of network observable related metrics. The labels in these metrics contain, among others:
```
Comment (sheriff-rh) [inline release_notes/ocp-4-6-release-notes.adoc:490]: network-observable, perhaps?

File: release_notes/ocp-4-6-release-notes.adoc [hunk 5]
```diff
@@ -453,6 +482,28 @@ By default, the following network connections are checked:
...
+
+[id="ocp-4-6-cnf-tests-discovery-mode"]
+==== CNF tests can be run in discovery mode
+
+There is an optional mode where the Cloud-native Network Functions (CNF) tests try to look for configurations on the cluster instead of applying the new ones. The CNF tests image is a containerized version of the CNF conformance test suite. It is intended to be run against a CNF-enabled {product-title} cluster where all the components required for running CNF workloads are installed.
+
+The tests must perform an environment configuration every time they are executed. This involves items such as creating SRIOV Node Policies, Performance Profiles, or PtpProfiles. Allowing the tests to configure an already configured cluster might affect the functionality of the cluster. Also, changes to configuration items such as SRIOV Node Policy might result in the environment being temporarily unavailable until the configuration change is processed.
```
Comment (sheriff-rh) [inline release_notes/ocp-4-6-release-notes.adoc:503]: We have this documented elsewhere as [SR-IOV](https://docs.openshift.com/container-platform/4.5/virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.html#discover-sr-iov-devices_virt-configuring-sriov-device-for-vms)

**Review by sheriff-rh — COMMENTED** [review]
This looks really great - just a few nits.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 5 above)
Comment (ahardin-rh) [reply release_notes/ocp-4-6-release-notes.adoc:503]: Great catch!

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (ahardin-rh) [reply release_notes/ocp-4-6-release-notes.adoc:430]: :sweat_smile: nice catch!

**Comment by ahardin-rh** [comment]
I'll add additional links to relevant topics in a separate PR once that content lands. Merging!

**Pull request merged**